### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by unnamedd**
## Description

A small improvement for the repo to ignore the changes in the auto generated files and adding a new section to the contributing.md file.

## Type of change

- [x] Documentation update

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+commands/README.md          -diff -merge
+commands/README.md          linguist-generated=true
```
Comment (shpakovski) [inline .gitattributes:2]: This one is new to me, thank you so much for introducing!

**Review by thomaspaulmann — APPROVED** [review]
Top! This should make it clearer for new contributors on what they have to do.

File: .gitattributes (hunk 1 above)
Comment (unnamedd) [reply .gitattributes:2]: @shpakovski thanks @thomaspaulmann for that, he found the link where these things were explained, I just brought it to life! 

I also didn't know exactly what the `.gitattributes` before!

Comment (shpakovski) [reply .gitattributes:2]: @thomaspaulmann 🤯🔥🙏

**Pull request merged**